snakefile.py

```diff
@@ -333,18 +333,111 @@ def tool(name):
     log: os.path.join(LOG_DIR, 'get_qc_table_{sample}.log')
     shell: "{PYTHON_EXEC} {params.script} {input.coverage_csv} {input.amplicon_csv} {output} >> {log} 2>&1"
         
-rule qc_report:
+
+#done by rendercite
+# rule qc_report:
+#     input:
+#         coverage_csv = os.path.join(COVERAGE_DIR, '{sample}_merged_covs.csv'),
+#         report_file = os.path.join(SCRIPTS_DIR, "qc_report_per_sample.rmd")
+#     output: os.path.join(REPORT_DIR, '{sample}_qc_report.html')
+#     params:
+#         sample_dir = VARIANTS_DIR,
+#         loc_sigmuts = SIGMUT_DB
+#     log: os.path.join(LOG_DIR, 'variant_report_{sample}.log')
+#     shell:
+#         """
+#         {RSCRIPT_EXEC} {SCRIPTS_DIR}/run_QC_report.R --reportFile={input.report_file} \
+#         --coverage_file={input.coverage_csv} --sample_name={wildcards.sample} \ 
+#         --outFile={output} >> {log} 2>&1
+#         """
+
+
+#added by mfaxel based on Boras snakemake.py from crispr_DART
```

Comment:
These comments are not very useful.  Attribution can be done via commit authorship and `Co-authored-by` lines in commit messages.